### PR TITLE
Add support for swing configuration

### PR DESCRIPTION
**Pull request by fabiomsouto**
Hey again,

Iterate a bit more on your awesome lib. I've added support for swing configuration to then carry it over to the hass component. 🎉 

Cheers.

### CODE REVIEW

File: wideq/ac.py [hunk 1]
```diff
@@ -4,6 +4,28 @@
 
 from .client import Device
 
+class ACVSwingMode(enum.Enum):
+    """The vertical swing mode for an AC/HVAC device."""
+    OFF = "@OFF"
```
Comment (fabiomsouto) [inline wideq/ac.py]: Blades are numbered vertically from 1 (topmost) to 6.

File: wideq/ac.py [hunk 2]
```diff
@@ -4,6 +4,28 @@
...
+    THREE = "@3"
+    FOUR = "@4"
+    FIVE = "@5"
+    SIX = "@6"
+    ALL = "@100"
+
+class ACHSwingMode(enum.Enum):
+    """The horizontal swing mode for an AC/HVAC device."""
+    OFF = "@OFF"
+    ONE = "@1"
```
Comment (fabiomsouto) [inline wideq/ac.py]: Blades are numbered horizontally from 1 (leftmost) to 5.

File: wideq/ac.py (hunk 1 above)
Comment (sampsyo) [reply wideq/ac.py]: Cool! Maybe it would be nice to include this information in a comment or the docstring for these enums?

**Comment by sampsyo** [comment]
Also, I'm adding you as a collaborator on this repository, so please feel free to merge your own pull request. :smiley:

**Comment by fabiomsouto** [comment]
Wow very honored :) 
I'll add these notes to the docs and merge then. Cheers

**Pull request merged**